packages/storybook/stories/AdditionalInfo.stories.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import {AdditionalInfo} from '@department-of-veterans-affairs/component-library';
+import { AdditionalInfo } from '@department-of-veterans-affairs/component-library';
 
 export default {
-  title: 'Components/AdditionalInfo',
+  title: 'Components/AdditionalInfo (deprecated)',
```

Comment:
Doing this will change the URL for the story that's [used on the component documentation page](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/blob/7744fe91a1501f07622f7a04930b88c35ae305df/src/_components/additional-info.md?plain=1#L14), so we may want to not include this or just immediately update design.va.gov to point to the Web Component story.